cord19-explorer/src/main/java/edu/asu/diging/cord19/explorer/web/ResultsController.java

```diff
@@ -0,0 +1,57 @@
+package edu.asu.diging.cord19.explorer.web;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.LongStream;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.diging.cord19.explorer.core.model.impl.PublicationImpl;
+import edu.asu.diging.cord19.explorer.core.mongo.PublicationSearchProvider;
+
+@Controller
+public class ResultsController {
+
+    @Autowired
+    private PublicationSearchProvider pubSearchProvider;
+
+    @RequestMapping(value = "/result")
+    public String search(@RequestParam("search") String title, Model model) {
+        long resultSize = pubSearchProvider.searchResultSize(title);
```

Comment:
and the variable name should be searchTerm or query or something. "title" sounds like it would be a complete title, which it isn't.